### PR TITLE
Docker Desktop 3.0.3 release notes

### PR DESCRIPTION
**Pull request by ebriney**
fixes https://github.com/docker/docker.github.io/issues/11965

### CODE REVIEW

**Comment by qhenkart** [comment]
3.0.3 broke https://github.com/docker/for-mac/issues/5124 which was fixed in 3.0.2.  The steps in the issue can be easily replicated to show the fsnotify event DELETE firing in every container when another container is mounted

**Comment by thaJeztah** [comment]
@stephen-turner @StefanScherer PTAL

Should we add https://github.com/docker/docker.github.io/pull/11956#issuecomment-751856085 to the "known issues" in this PR?

File: docker-for-mac/release-notes.md [hunk 1]
```diff
@@ -16,10 +16,23 @@ This page contains information about the new features, improvements, known issue
...
 {: .important }
 
+## Docker Desktop Community 3.0.3
+2020-12-21
+
+> [Download](https://hub.docker.com/editions/community/docker-ce-desktop-mac/)
+
+### Bug fixes and minor changes
+
+- Rollback gprcfuse patches that fixed [docker/for-mac#4957](https://github.com/docker/for-mac/issues/4957) and [docker/for-mac#5124](https://github.com/docker/for-mac/issues/5124). They broke widely used mount configurations. See [docker/for-mac#5157](https://github.com/docker/for-mac/issues/5157).
```
Comment (stephen-turner) [inline docker-for-mac/release-notes.md:26]: I think the sentence needs to be in the opposite order:

- Fixed an issue that caused overlapping volume mounts to fail. Fixes [docker/for-mac#5157](https://github.com/docker/for-mac/issues/5157). However, the fixes for [docker/for-mac#4957](https://github.com/docker/for-mac/issues/4957) and [docker/for-mac#5124](https://github.com/docker/for-mac/issues/5124) have been reverted as a result of this change, so those issues are now present again.

File: docker-for-mac/release-notes.md [hunk 2]
```diff
@@ -16,10 +16,23 @@ This page contains information about the new features, improvements, known issue
...
+
+### Known issues
+
+- Some DNS addresses fail to resolve within containers based on Alpine Linux 3.13. See [docker/for-mac#5020](https://github.com/docker/for-mac/issues/5020).
+
```
Comment (stephen-turner) [inline docker-for-mac/release-notes.md:31]: Add:

- There can be timeouts during `docker-compose up` if there are several services being started. See [docker/for-mac#4957](https://github.com/docker/for-mac/issues/4957) and [docker/for-mac#5124](https://github.com/docker/for-mac/issues/5124).

**Review by stephen-turner — CHANGES_REQUESTED** [review]

**Comment by thaJeztah** [comment]
@ebriney can you address @stephen-turner's comments?

**Review by stephen-turner — APPROVED** [review]
Thanks, looks good now.

**Pull request merged**